### PR TITLE
feat: implement util.whois

### LINKED ISSUE #2: util.whois: Get basic info of Telegram accounts
The whois command displays the information of the replied peer or the passed peer ID.

The information can be received with the `getEntity` method of `TelegramClient`.

### PR DESCRIPTION
**Pull request by ArnabXD**
- Closes #2.

### CODE REVIEW

**Comment by ArnabXD** [comment]
IMG_20220129_113000

**Comment by ArnabXD** [comment]
I think we need to be consistent about using Unicode symbols. What symbol would you prefer @roj1512?
I used `&#9187;` (`⏣`) in this module.

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:47]:
```suggestion
							: 'N/A'
```

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:44]:
```suggestion
					`<b>DC:</b> ${
```

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:43]:
```suggestion
					`<b>ID:</b> <code>${user.id}</code>\n` +
```

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:40]:
```suggestion
					`<a href="tg://user?id=${user.id}"><b>${escape(
```

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:49]:
```suggestion
					`<b>Username:</b> ${user.username ? '@' + user.username : 'N/A'}\n` +
```

File: src/modules/whois.ts
Comment (roj1512) [inline src/modules/whois.ts:50]:
```suggestion
					`<b>Description:</b> ${fullUser.about || 'N/A'}`,
```

**Review by roj1512 — CHANGES_REQUESTED** [review]
I don't like the response format. Maybe we can stay a little minimal or something?

Also, can we remove those as statements in some matter?

**Comment by ArnabXD** [comment]
> Also, can we remove those as statements in some matter?

non-null assertion is forbidden by `eslint@typescript-eslint/no-non-null-assertion`.

We need to disable this to reduce `as`.

**Comment by roj1512** [comment]
No need to disable that rule. All of the `as`s can be removed too. For example, for the replied message you can do something like this:
```ts
const reply = await event.message.getReplyMessage();
if (reply) // use
```
Which is even more appropriate and does your current job.

**Comment by ArnabXD** [comment]
looks fine to me

**Comment by roj1512** [comment]
@ArnabXD wait until I approve.

**Review by roj1512 — APPROVED** [review]

**Pull request merged**
